pkgs/README.md

```diff
@@ -30,12 +30,28 @@ Before adding a new package, please consider the following questions:
 
 * Is the package ready for general use? We don't want to include projects that are too immature or are going to be abandoned immediately. In case of doubt, check with upstream.
 * Does the project have a clear license statement? Remember that software is unfree by default (all rights reserved), and merely providing access to the source code does not imply its redistribution. In case of doubt, ask upstream.
-* How realistic is it that it will be used by other people? It's good that nixpkgs caters to various niches, but if it's a niche of 5 people it's probably too small.
+* How realistic is it that it will be used by other people? It's good that nixpkgs caters to various niches, but if it's a niche of 5 people it's probably too small. A good estimate is checking upstream issues and pull requests, or other software repositories.
+* Is the software maintained upstream? Especially security-critical components should see regular maintenance.
```

Comment:
```suggestion
* Is the software maintained upstream? Things at high risk of breakage, in security or dependency compatibility or data integrity sense, should see regular maintenance.
```

I know the PR is about security, but the question added is not just about security (and unsafe even without attackers is no better!)